src/components/Collapse/Collapse.test.ts

```diff
@@ -7,4 +8,12 @@ describe('template', () => {
     });
 
     componentSlotRenderTest(Collapse);
+
+    it('emits event on click collapse toggle', async() => {
+        const wrapper = shallowMount(Collapse);
+
+        wrapper.findComponent(Collapse).vm.$emit('click:toggle');
+
+        expect(wrapper.emitted('click:toggle')).toBeTruthy();
```

Comment:
This test will never ever fail. It is like giving the answers of a test to a student. You need to test that the `click` event is resulting in the `click:toggle` event.